behat.yml.dist

```diff
@@ -6,14 +6,12 @@ default:
       contexts:
         - Drupal\DrupalExtension\Context\MinkContext
         - Drupal\DrupalExtension\Context\DrupalContext
-        - Drupal\Tests\oe_corporate_blocks\Behat\CorporateBlocksContext:
-            selectors:
-              site switcher active link: '.site-switcher a.active'
         - Drupal\DrupalExtension\Context\ConfigContext
+        - Drupal\Tests\oe_corporate_blocks\Behat\CorporateBlocksContext:
+          - selector: ~
```

Comment:
Probably we shouldn't keep anymore this argument and it could be removed?:
```suggestion
```

Reply:
Removed.